platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/activity.junit.ejs

```diff
@@ -27,7 +27,7 @@ public class <%- activity %>Test extends BaseActivityTest {
 
     @Test
     public void testSanity() {
-        validateTestSetup();
+        // no-op
```

Comment:
Can we remove this test all-together?

Reply:
For now I would keep the sanity tests around, updated above block however with assertions for mapboxMap, style and fully loaded.